package.json

```diff
@@ -26,7 +26,9 @@
     "async": "^1.5.0",
     "buffers": "^0.1.1",
     "chalk": "^1.1.1",
+    "clone": "~1.0.2",
     "gulp-util": "^3.0.3",
+    "lodash": "~4.11.0",
```

Comment:
Instead of require all lodash library can you select only lodash.assign (and lodash.isObject if utils really fail)?